README.md

```diff
@@ -1,77 +1,150 @@
+| `COUNTDOWN` | The current state of verification. | Detector: `verification_detector`<br/>Labels: `verification_labels`<br/>Confidence: `verification_confidence` | `DISARMED` (on trigger) / `ALARM` (on timeout) |
+| `ALARM`    | The alarm state. | - | TRIGGER_1 (after `alarm_time_s`) |
+| `DISARMED` | The disarmed state.| - | TRIGGER_1 (after `disarmed_time_s`) |
 
-The module sets up a state machine with 5 states:
-1. `TRIGGER_1`: The module begins in this state. It is meant to be a coarse, fast detector, like a simple motion detector. This state runs the `trigger_1_detector` on every frame, looking for detections with any label from `trigger_1_labels` with at least `trigger_1_confidence`. If the detector triggers, then the state moves to `TRIGGER_2`. If no `TRIGGER_1` detector was specified in the config, the module moves immediately to state `TRIGGER_2`.
-2. `TRIGGER_2`: This state runs the `trigger_2_detector` on every frame, looking for detections with any label from `trigger_2_labels` with at least `trigger_2_confidence`. If the detector triggers, then the state moves to `COUNTDOWN`. If it doesn't trigger in 10 frames, it returns to state `TRIGGER_1`.
-3. `COUNTDOWN`: This state runs the `verification_detector` on every frame, looking for detections with any label from `verification_labels` with at least `verification_confidence`. If the detector triggers, then the state moves to `DISARMED`. If it doesn't trigger in the time specified by `countdown_time_s`, it moves to state `ALARM`.
-4. `ALARM`: The alarm state. The module will emit the `ALARM` classification for the amount of time specified in `alarm_time_s`. After that amount of time elapses, the module will return to state `TRIGGER_1`.
-5. `DISARMED`: The disarmed state. The module will emit the `DISARMED` classification for the amount of time specified in `disarmed_time_s`. After that amount of time elapses, the module will return to state `TRIGGER_1`.
+## Build and Run
 
-If you do not want the `ALARM` capabilities, and would like to just use it as a notification system when a detector gets triggered, you can set `disable_alarm: true` in the config, which will prevent `TRIGGER_2` from entering into the `COUNTDOWN` state. This means the system will only cycle between the states of `TRIGGER_1` and `TRIGGER_2`. A user can use entering into the state `TRIGGER_2` as a way to send notifications.
+To use this module, configure your machine with the resources outlined in this readme.
 
-If you want to save images to the Viam cloud when the system enters into specific states, you can use the [filtered-camera module](https://app.viam.com/module/erh/filtered-camera) in tandem with this module.
+Before configuring the resources for this module, [Create a new machine](https://docs.viam.com/fleet/machines/#add-a-new-machine) in the Viam app. Then [Install `viam-server`](https://docs.viam.com/get-started/installation/) on your new machine.
```

Comment:
Can remove this and add "create a machine" and "Configure a Machine" notes from template